<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetSize</class>
 <widget class="QDialog" name="SetSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QDialog {
	background-image: url(:/Images/Marble.jpg);
}

QLabel{
font: bold &quot;Ubuntu&quot;;
font-size: 12px;
color: lightgray;
}

QRadioButton{
font:  bold &quot;Ubuntu&quot;;
font-size:11px;
color: lightgray;
}

QRadioButton::indicator::checked{
background: url(:/Images/Ressource/check-marks_check.png);
width: 33;
height: 26;
}

QRadioButton::indicator:unchecked{
background: url(:/Images/Ressource/check-marks_uncheck.png);
width: 33;
height: 26;
}

QPushButton {
color: #DFD;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,  stop: 0 #6B82AC, stop: 0.49 #566D97, stop: 0.5 #445B85, stop: 1 #566D97);
border-width: 2px;
border-color: #778EB8;
border-style: solid;
border-radius: 7;
padding: 3px;
font: bold &quot;Ubuntu&quot;;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
}

QLineEdit{
background-color: lightgray;
border-width: 2px;
border-color: #778EB8;
border-style: solid;
border-radius: 7;
}

*/</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_480x272">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>480 x 272</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_320x200">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>320 x 200</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_240x320">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>240 x 320</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_800x600">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>800 x 600</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_640x480">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>640 x 480</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_128x128">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>128 x 128</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_CustomSize">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom size</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_96x64">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>96 x 64</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CustomWidth">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_800x480">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>800 x 480</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_320x240">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>320 x 240</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWidth">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Height">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CustomHeight">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_84x48">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>84 x 48</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display Size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetSize</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetSize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
